src/ParametricOptInterface.jl

```diff
@@ -180,6 +189,14 @@ function MOI.supports_constraint(
     )
 end
 
+function MOI.supports_constraint(
+    model::Optimizer,
+    F::Type{MOI.VectorQuadraticFunction{T}},
+    S::Type{<:MOI.AbstractSet},
+) where {T}
+    return MOI.supports_constraint(model.optimizer, F, S)
+end
```

Comment:
The inner optimizer does not need to support quadratic. Because the quadratic input might be parameter*variable